system/cfme-setup.sh

```diff
@@ -40,9 +40,6 @@ EOF
 [ -x /sbin/restorecon ] && /sbin/restorecon -R -v /var/www/miq/vmdb/certs
 [ -x /sbin/restorecon ] && /sbin/restorecon -R -v /var/www/miq/system/logrotate_free_space_check.sh
 
-# relabel the pg_log directory in postgresql datadir, but defer restorecon
-# until after the database is initialized during firstboot configuration
-/usr/sbin/semanage fcontext -a -t var_log_t "/opt/rh/postgresql92/root/var/lib/pgsql/data/pg_log(/.*)?"
```

Comment:
@kbrock I don't understand the details on what this `semanage` WAS doing vs. what the change you made to do `restorecon` on the whole DATABASE_DISK_MOUNT_POINT vs. just the log directory.